tests/network/probes.go

```diff
@@ -71,16 +71,15 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 		isVMIReady := func() bool {
```

Comment:
This function results is being asserted against as a boolean.
I find this problematic, as in case of a failure, it will reveal no useful information of what was expected.

If on the other hand, this would have been returning the condition status, then the assertion would fail with a more informative information.

Reply:
ok, I've created `getVMIConditions()` to be more descriptive on failure.